### ISSUE TITLE
Google Play Services(and GFS) background data enabled automatically upon reboot

### ISSUE THREAD

**Issue by jayb-g** [GrapheneOS/os-issue-tracker#3613]
One, I see that GPS and GFS permissions are mirrored automatically to each other whenever changed in either of them. Not sure if its expected behavior.

Second, if GPS/GFS permission is set to disable Background Data, it gets enabled automatically if phone is rebooted. I don't think this would be an expected behavior.

Steps to reproduce:
Disable Background Data for GPS/GFS and reboot the phone. Check the permissions after reboot to find it enabled again automatically.

**Comment by jayb-g** [comment]
How so?

**Comment by 5atoshiNakamoto** [comment]
Yes, the "allow background data" setting on apps being set back to on after a reboot or just changing users is a big flaw, particularly when one is trying to limit their data usage.
